.github/workflows/deploy-studies-bridgedigitalhealth-staging.yaml

```diff
@@ -12,5 +12,5 @@ jobs:
     uses: ./.github/workflows/build-and-deploy.yaml
     with:
       branch: ${{ github.ref_name }}
-      destination_bucket: tst-bridgedigitalhealth-websitebucket-1fnzdxhja8yl4
-      distribution_id: E86GBF8L8W74U
\ No newline at end of file
+      destination_bucket: staging-studies-bridgedigital-healt-websitebucket-llv54doyeqrl
```

Comment:
The name of the bucket is actually created by Cloudformation in this case (taking the name of the stack as prefix). Some operations need to replace a resource, so they recommend not naming them. In this case, I'm not sure it's best since we have a dependency on the name in the action (not too worried, once things settle down it likely won't change)...